### PR TITLE
A quick example to set up a monitor on a live stream

### PR DESCRIPTION
**Pull request by paulina-positronix**

### CODE REVIEW

File: docs/docs/getting-started/5-streaming.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+Ensure you have Python 3.7 or higher installed, and then install the Groundlight SDK and OpenCV library:
+
+```bash
+pip install groundlight pillow ffmpeg yt-dlp typer
+```
+
+## Creating the Application
+
+1. Save this command as a shell script `get_last_frame.sh`:
```
Comment (tyler-romero) [inline docs/docs/getting-started/5-streaming.md:20]: nit: `get_latest_frame.sh` might be more clear

File: docs/docs/getting-started/5-streaming.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# A Quick Example: Live Stream Alert 
+
+A quick example to get used to setting up detectors and asking good questions: set up a monitor on a live stream. 
+
+## Requirements
+
+- [Groundlight SDK](/docs/installation/) with Python 3.7 or higher
+- The video ID of a YouTube live stream you'd like to monitor
```
Comment (tyler-romero) [inline docs/docs/getting-started/5-streaming.md:8]: How does the user identify a video id for a live stream?

File: docs/docs/getting-started/5-streaming.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+3. Next, we'll write the Python script for the application.
+
+```python notest
+import os
+import subprocess
+import typer
+from groundlight import Groundlight
+from PIL import Image
+
+GROUNDLIGHT_API_TOKEN=os.getenv("GROUNDLIGHT_API_TOKEN")
```
Comment (tyler-romero) [inline docs/docs/getting-started/5-streaming.md:41]: Should we include instructions for the user to set this env var? or is that too basic?

File: docs/docs/getting-started/5-streaming.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+
+def main(*, video_id: str = None, detector_name: str = None, query: str = None, confidence: float = 0.75, wait: int = 60):
+    """
+    Run the script to get the stream's last frame as a subprocess, and submit result as an image query to a Groundlight detector
+    :param video_id: Video ID of the YouTube live stream
+    :param detector_name: Name for your Groundlight detector
+    :param query: Question you want to ask of the stream (we will alert on the answer of NO)
+    """
+    gl = Groundlight(api_token=GROUNDLIGHT_API_TOKEN)
```
Comment (tyler-romero) [inline docs/docs/getting-started/5-streaming.md:51]: It should automatically be set inside of `Groundlight` already, but if this is just to be illustrative, that is ok

**Review by tyler-romero — APPROVED** [review]

File: docs/docs/getting-started/5-streaming.md (hunk 3 above)
Comment (paulina-positronix) [reply docs/docs/getting-started/5-streaming.md:41]: I'm going to remove this since it's set automatically inside groundlight

**Pull request merged**